src/clients/inst_setup_dhcp.rb

```diff
@@ -1,24 +1,3 @@
-require "yast"
-require "network/network_autoconfiguration"
-
-module Yast
-  class SetupDhcp
-    include Singleton
-    include Logger
-
-    def main
-      nac = Yast::NetworkAutoconfiguration.instance
-      if !nac.any_iface_active?
-        nac.configure_dhcp
-      else
-        log.info("Automatic DHCP configuration not started - an interface is already configured")
-      end
-
-      # if this is not wrapped in a def, ruby -cw says
-      # warning: possibly useless use of a literal in void context
-      :next
-    end
-  end
-end
```

Comment:
What you mean? I don't get you. I just moved the client to lib, but no take any decision about it.